ReactWindows/ReactNative/Chakra/Executor/NativeJavaScriptExecutor.cs

```diff
@@ -1,37 +1,65 @@
-﻿using Newtonsoft.Json;
+using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 using ReactNative.Bridge;
 using System;
 using System.IO;
+using System.Threading.Tasks;
 using Windows.Storage;
 
 namespace ReactNative.Chakra.Executor
 {
+    /// <summary>
+    /// Tells how to serialize the JS bundle to bytecode when the app starts.
+    /// </summary>
+    public enum SerializationMode
+    {
+        /// <summary>
+        /// Load the JS bundle without serialization.
+        /// </summary>
+        None,
+
+        /// <summary>
+        /// Serialize the bundle to bytecode first, then load the bundle.
+        /// Next time the app starts it will load the existing bytecode.
+        /// </summary>
+        Foreground,
```

Comment:
Do we even need this if Background serialization is an option? Is it ever really a good idea to wait for the bundle to serialize?

Reply:
I don't think we need this option. I added it just in case.